java/spacewalk-java.spec

```
@@ -549,8 +549,8 @@ install -m 644 conf/rhn_java_sso.conf %{buildroot}%{_datadir}/rhn/config-default
 # Adjust product tree tag
 %if 0%{?is_opensuse}
 sed -i -e 's/^java.product_tree_tag =.*$/java.product_tree_tag = Uyuni/' %{buildroot}%{_datadir}/rhn/config-defaults/rhn_java.conf
-%else
-sed -i -e 's/^java.product_tree_tag =.*$/java.product_tree_tag = Beta/' $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_java.conf
+##%else
```

we need a double `%` because the macros are expanded also  in comments. and  maybe,   if you want to comment the code instead  of dropping it, we could add a comment that explains why
```suggestion
##%%else
```

and I think we should do  the  same also below, because that `%{_prefix}` will be expanded

---

I have removed the lines. Than it is clea